src/AppBundle/Command/AddUserCommand.php

```diff
@@ -193,6 +187,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $encoder = $this->getContainer()->get('security.password_encoder');
         $encodedPassword = $encoder->encodePassword($user, $plainPassword);
         $user->setPassword($encodedPassword);
+        $user->setPlainPassword($plainPassword);
+
+        // See http://symfony.com/doc/current/book/validation.html
+        $errors = $this->validator->validate($user, null, array('Default', 'registration'));
+
+        if (count($errors) > 0) {
```

Comment:
According to [Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html) expected value should be first: `if (0 < count($errors)) {`